docs/docson/build-schema.json

```diff
@@ -185,6 +185,31 @@
             ],
             "description": "can be either a single sourceItem or an array of sourceItems, sourceItem can be either a single string(non nested directory) or an detailed object config"
         },
+        "targetItems": {
+          "type": "object",
+          "properties": {
+            "outputName": {
+              "type": "string",
+              "description": "The name of the target"
+            },
+            "compilationKind": {
+              "enum": [
+                "js",
+                "native",
+                "bytecode"
+              ],
+              "description": "The compiler to use for the target"
+            },
+            "entryPoint": {
+              "type": "string",
```

Comment:
entryPoint only makes sense for native backend, can we make the schema enforce such invariant?